connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -201,6 +201,20 @@ class SparkConnectService(debug: Boolean) extends AsyncService with BindableServ
         sessionId = request.getSessionId)
   }
 
+  override def getErrorInfo(
+      request: proto.GetErrorInfoRequest,
+      responseObserver: StreamObserver[proto.GetErrorInfoResponse]): Unit = {
+    try {
+      new SparkConnectGetErrorInfoHandler(responseObserver).handle(request)
+    } catch {
+      ErrorUtils.handleError(
+        "getErrorInfo",
+        observer = responseObserver,
+        userId = request.getUserContext.getUserId,
+        sessionId = request.getSessionId)
```

Comment:
The idea is to make the ErrorInfo self-contained for issuing FetchErrorDetails request. Otherwise, it may make the implementation on the client side a bit more complicate since each FetchErrorDetails call needs to figure out the corresponding session id and user id